plugin.video.carnet-meduza/resources/lib/carnet-meduza.py

```diff
@@ -0,0 +1,270 @@
+			description = video_info['opis'].encode('utf-8')
+			li = xbmcgui.ListItem(name)
+			li.setArt({'icon':image})
+			li.setInfo( type="Video", infoLabels={ 
+								"Plot": description, 
+								"Duration": duration,
+								"mediatype":"video"
+								})
+			xbmcplugin.addDirectoryItem(handle=addon_handle, url=url, listitem=li)
+		pages_num = video_num / 25
+		if int(current_page_number) < pages_num:
+			int_current_page_number = int(current_page_number)
+			int_current_page_number += 1
+			current_page_number = str(int_current_page_number)
+			url = build_url({'mode': 'folder', 'foldername': foldername, 'pagenumber': current_page_number})
+			li = xbmcgui.ListItem('> Next Page (' + str(int_current_page_number + 1) + ')', iconImage="DefaultFolder.png")
+			xbmcplugin.addDirectoryItem(handle=addon_handle, url=url, listitem=li, isFolder=True)		
+		xbmcplugin.endOfDirectory(addon_handle)
+		
+def start_channel(channel_id,channel_video_count):
+	playlist = xbmc.PlayList(xbmc.PLAYLIST_VIDEO)
+	player = xbmc.Player()
+	playlist.clear()
+	requestChannel = requests.get(api_base_url + 'channel/?id=' + channel_id + '&uid=' + api_key)
+	channel = requestChannel.json()
+	channel_index = channel['index']
+	schedule = channel['raspored']
+	channel_offset = channel['offset']
+	schedule_gen = ((i, j) for i, j in enumerate(schedule))
+	for i, j in schedule_gen:
+	        li = xbmcgui.ListItem(j['naslov'])
+		li.setArt({'thumb':j['slika']})
+		itemArgs = {
+                'title': j['naslov'].encode('utf-8'),
+                'plot': j['opis'].encode('utf-8'),
+		'mediatype':'video',
+                'tracknumber': i 
+            	}
+		li.setInfo('video', itemArgs)
+		playlist.add(url=j['stream_url'],listitem=li)
+        player.play(item=playlist,startpos=channel_index)
+	player.seekTime(channel_offset)
+
+base_url = sys.argv[0]
+addon_handle = int(sys.argv[1])
+args = urlparse.parse_qs(sys.argv[2][1:])
+#set as video addon
+xbmcplugin.setContent(addon_handle, 'videos')
+
+addon=xbmcaddon.Addon('plugin.video.carnet-meduza')
+api_base_url = 'https://meduza.carnet.hr/index.php/api/'
+category_image_base_url = 'https://meduza.carnet.hr/uploads/images/categories/'
+# api_key == device_id from register (import) device == uid
+api_key = addon.getSetting('apikey')
+
+mode = args.get('mode', None)
+
+dir_recommends = addon.getLocalizedString(30201)
+dir_categories = addon.getLocalizedString(30202)
+dir_channels = addon.getLocalizedString(30203)
+dir_search = addon.getLocalizedString(30204)
+dir_settings = addon.getLocalizedString(30205)
+
+# get api lang property (en|hr)
+active_lang = xbmc.getLanguage()
+if active_lang == 'Croatian':
+	lang_api_prop = 'naziv'
+else:
+	lang_api_prop = 'naziv_en'
+
+if mode is None:
+    url = build_url({'mode': 'folder', 'foldername': dir_recommends})
+    li = xbmcgui.ListItem(dir_recommends)
+    li.setArt({'icon':'DefaultVideo.png'})
+    xbmcplugin.addDirectoryItem(handle=addon_handle, url=url, listitem=li, isFolder=True)
```

Comment:
What are my alternatives. According to http://mirrors.kodi.tv/docs/python-docs/16.x-jarvis/xbmcgui.html#ListItem-setInfo, offered are
"video", "movie", "tvshow", "season", "episode" or "musicvideo", where video is most suitable for the content being offered.